.gitignore

```diff
@@ -34,3 +34,7 @@ src/data/attachments/*
 !src/data/attachments/index.php
 src/data/attachments/deleted/*
 !src/data/attachments/deleted/index.php
+
+# Custom logos directory
```

Comment:
I think is better if you make the custom icons directory in ```src/data/customlogos``` and make sure you create that directory in the provision script and make it world writable: https://github.com/facebook/fbctf/blob/master/extra/provision.sh#L252